10-22 18:04:28 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAY_EVOLUTIONS" not found; SQL statement:
select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%' [42102-168]
10-23 09:41:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE *[*] FROM USERS"; expected "identifier"; SQL statement:
DELETE * FROM USERS [42001-168]
10-23 09:43:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM USERS_SECURITY_ROLE 

DELETE FROM[*] USERS_SECURITY_ROLE WHERE ID = 33"; SQL statement:
SELECT * FROM USERS_SECURITY_ROLE 

DELETE FROM USERS_SECURITY_ROLE WHERE ID = 33 [42000-168]
10-23 09:43:27 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "ID" not found; SQL statement:
DELETE FROM USERS_SECURITY_ROLE WHERE ID = 33 [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:138)
	at org.h2.expression.Comparison.optimize(Comparison.java:159)
	at org.h2.command.dml.Delete.prepare(Delete.java:125)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1109)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:967)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:724)
10-23 09:43:39 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "USER_ID" not found; SQL statement:
DELETE FROM USERS_SECURITY_ROLE WHERE USER_ID = 33 [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:138)
	at org.h2.expression.Comparison.optimize(Comparison.java:159)
	at org.h2.command.dml.Delete.prepare(Delete.java:125)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1109)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:967)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:724)
10-23 09:52:34 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "USERS_ID" not found; SQL statement:
DELETE FROM LINKED_ACCOUNT  WHERE USERS_ID = 67 [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:138)
	at org.h2.expression.Comparison.optimize(Comparison.java:159)
	at org.h2.command.dml.Delete.prepare(Delete.java:125)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1109)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:967)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:724)
03-19 11:23:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "USERS[*] SELECT * FROM LINKED_ACCOUNT"; expected "UPDATE, {"; SQL statement:
USERS SELECT * FROM LINKED_ACCOUNT [42001-168]
03-19 15:41:11 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO AUTHTOKEN ('USER'[*], 'PROVIDER', 'TOKEN') VALUES ('?', '?','?');"; expected "identifier"; SQL statement:
INSERT INTO AUTHTOKEN ('USER', 'PROVIDER', 'TOKEN') VALUES ('?', '?','?'); [42001-168]
03-19 15:41:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO AUTHTOKEN ('USER'[*], 'PROVIDER', 'TOKEN') VALUES ('?', '?','?');"; expected "identifier"; SQL statement:
INSERT INTO AUTHTOKEN ('USER', 'PROVIDER', 'TOKEN') VALUES ('?', '?','?'); [42001-168]
03-19 15:43:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO AUTHTOKEN ('USER'[*], 'PROVIDER', 'TOKEN') VALUES ('?', '?','?');"; expected "identifier"; SQL statement:
INSERT INTO AUTHTOKEN ('USER', 'PROVIDER', 'TOKEN') VALUES ('?', '?','?'); [42001-168]
03-19 15:45:10 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO AUTHTOKEN ('USER'[*], 'PROVIDER', 'TOKEN') VALUES ('?', '?','?');"; expected "identifier"; SQL statement:
INSERT INTO AUTHTOKEN ('USER', 'PROVIDER', 'TOKEN') VALUES ('?', '?','?'); [42001-168]
03-19 15:50:59 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO AUTHTOKEN ('USER'[*], 'PROVIDER', 'TOKEN') VALUES ('1', 'facebook','1')"; expected "identifier"; SQL statement:
INSERT INTO AUTHTOKEN ('USER', 'PROVIDER', 'TOKEN') VALUES ('1', 'facebook','1') [42001-168]
03-19 15:55:02 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "1" for parameter "parameterIndex" [90008-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1284)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:340)
	at com.jolbox.bonecp.PreparedStatementHandle.setString(PreparedStatementHandle.java:1029)
	at models.AuthToken.updateStatus(AuthToken.java:55)
	at controllers.Application.restricted(Application.java:49)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$6$$anonfun$apply$6.apply(routes_routing.scala:217)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$6$$anonfun$apply$6.apply(routes_routing.scala:217)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:57)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:44)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:99)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
03-19 15:59:29 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "1" for parameter "parameterIndex" [90008-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1284)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:340)
	at com.jolbox.bonecp.PreparedStatementHandle.setString(PreparedStatementHandle.java:1029)
	at models.AuthToken.updateStatus(AuthToken.java:55)
	at controllers.Application.restricted(Application.java:49)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$6$$anonfun$apply$6.apply(routes_routing.scala:217)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$6$$anonfun$apply$6.apply(routes_routing.scala:217)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:57)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:44)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:99)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
03-19 16:01:51 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "1" for parameter "parameterIndex" [90008-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1284)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:340)
	at com.jolbox.bonecp.PreparedStatementHandle.setString(PreparedStatementHandle.java:1029)
	at models.AuthToken.updateStatus(AuthToken.java:55)
	at controllers.Application.restricted(Application.java:49)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$6$$anonfun$apply$6.apply(routes_routing.scala:217)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$6$$anonfun$apply$6.apply(routes_routing.scala:217)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:57)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:44)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:99)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
03-19 16:09:09 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "AUTHTOKEN" not found; SQL statement:
SELECT * FROM AUTHTOKEN WHERE USER=? AND PROVIDER=?  [42102-168]
03-19 16:10:00 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "AUTHTOKEN" not found; SQL statement:
SELECT * FROM AUTHTOKEN WHERE USER=? AND PROVIDER=?  [42102-168]
03-26 10:54:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FACEBOOK" not found; SQL statement:
INSERT INTO AUTHTOKEN (USER, PROVIDER, TOKEN) VALUES (1,facebook,jhbjhb) [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:138)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1109)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:967)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:744)
03-26 10:55:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO AUTHTOKEN (ID, USER, PROVIDER, TOKEN) VALUES (,[*]1,2,3)"; expected "), DEFAULT, NOT, EXISTS, SELECT, FROM"; SQL statement:
INSERT INTO AUTHTOKEN (ID, USER, PROVIDER, TOKEN) VALUES (,1,2,3) [42001-168]
03-26 15:32:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "USERS[*] SELECT * FROM USERS"; expected "UPDATE, {"; SQL statement:
USERS SELECT * FROM USERS [42001-168]
03-26 17:06:37 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Column "business_dimensions" not found [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at models.User.retrieveUser(User.java:257)
	at controllers.Application.retrieveUser(Application.java:109)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
03-26 17:06:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "business_dimensions" not found [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at models.User.retrieveUser(User.java:257)
	at controllers.Application.retrieveUser(Application.java:109)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
03-26 17:07:18 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "business_dimensions" not found [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at models.User.retrieveUser(User.java:257)
	at controllers.Application.retrieveUser(Application.java:109)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
03-26 17:09:14 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Column "business_dimensions" not found [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at models.User.retrieveUser(User.java:257)
	at controllers.Application.retrieveUser(Application.java:109)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
03-26 17:48:24 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "eyJhbGciOiJIUzI1NiIsImN0eSI6InRleHRcL3BsYWluIn0.MQ.XW8vbtoz5nEU3buRIquYT8ZAjNEKlW_GvLYVQxNXk2I"; SQL statement:
SELECT * FROM USERs WHERE ID=? [22018-168]
03-26 17:49:28 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "eyJhbGciOiJIUzI1NiIsImN0eSI6InRleHRcL3BsYWluIn0.MQ.XW8vbtoz5nEU3buRIquYT8ZAjNEKlW_GvLYVQxNXk2I"; SQL statement:
SELECT * FROM USERs WHERE ID=? [22018-168]
04-03 15:19:03 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "a"; SQL statement:
SELECT * FROM USERs WHERE ID=? [22018-168]
04-07 17:04:53 database: close
org.h2.message.DbException: IO Exception: "java.io.EOFException"; "/Users/Riboni1989/Desktop/Lavoro/LucaG/authentication/data/default.h2.db" [90031-168]
	at org.h2.message.DbException.get(DbException.java:158)
	at org.h2.message.DbException.convertIOException(DbException.java:315)
	at org.h2.store.FileStore.readFully(FileStore.java:278)
	at org.h2.store.PageStore.readPage(PageStore.java:1285)
	at org.h2.store.PageStore.getPage(PageStore.java:735)
	at org.h2.store.PageStreamTrunk$Iterator.next(PageStreamTrunk.java:247)
	at org.h2.store.PageLog.free(PageLog.java:210)
	at org.h2.store.PageStore.compact(PageStore.java:491)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1199)
	at org.h2.engine.Database.close(Database.java:1149)
	at org.h2.engine.DatabaseCloser.run(DatabaseCloser.java:80)
Caused by: org.h2.jdbc.JdbcSQLException: IO Exception: "java.io.EOFException"; "/Users/Riboni1989/Desktop/Lavoro/LucaG/authentication/data/default.h2.db" [90031-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 11 more
Caused by: java.io.EOFException
	at org.h2.store.fs.FileUtils.readFully(FileUtils.java:359)
	at org.h2.store.FileStore.readFully(FileStore.java:276)
	... 8 more
04-07 17:25:31 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Column "provider" not found [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at models.User.retrieveUser(User.java:267)
	at controllers.Application.retrieveUser(Application.java:56)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
04-07 17:27:55 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Column "provider" not found [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at models.User.retrieveUser(User.java:269)
	at controllers.Application.retrieveUser(Application.java:56)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
04-07 17:30:30 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Column "provider" not found [42122-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:2893)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2979)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:291)
	at models.User.retrieveUser(User.java:268)
	at controllers.Application.retrieveUser(Application.java:56)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$7$$anonfun$apply$7.apply(routes_routing.scala:229)
	at play.core.Router$HandlerInvoker$$anon$6$$anon$2.invocation(Router.scala:193)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:373)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:50)
	at play.GlobalSettings$1.call(GlobalSettings.java:63)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:80)
	at play.core.j.JavaAction$$anon$2.apply(JavaAction.scala:79)
	at play.libs.F$Promise$PromiseActor.onReceive(F.java:425)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:159)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:425)
	at akka.actor.ActorCell.invoke(ActorCell.scala:386)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:230)
	at akka.dispatch.Mailbox.run(Mailbox.scala:212)
	at akka.dispatch.ForkJoinExecutorConfigurator$MailboxExecutionTask.exec(AbstractDispatcher.scala:502)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:262)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:975)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1478)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104)
04-11 12:31:25 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "USERS[*] SELECT * FROM USERS"; expected "UPDATE, {"; SQL statement:
USERS SELECT * FROM USERS [42001-168]
04-11 12:47:57 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "USER_PERMISSION"; expected "UPDATE, {"; SQL statement:
USER_PERMISSION [42001-168]
04-11 12:48:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "USER_PERMISSION"; expected "UPDATE, {"; SQL statement:
USER_PERMISSION [42001-168]
04-11 16:04:15 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1384)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1359)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:305)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:622)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:677)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
04-11 16:04:15 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1384)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1359)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:305)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:622)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:677)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
04-11 16:04:15 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1384)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1359)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:305)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:622)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:677)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
04-11 16:04:15 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-168]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1384)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1359)
	at org.h2.jdbc.JdbcConnection.getMetaData(JdbcConnection.java:305)
	at com.jolbox.bonecp.ConnectionHandle.getMetaData(ConnectionHandle.java:622)
	at com.jolbox.bonecp.BoneCP.isConnectionHandleAlive(BoneCP.java:677)
	at com.jolbox.bonecp.ConnectionTesterThread.run(ConnectionTesterThread.java:103)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
05-09 10:15:10 jdbc[10]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 17 more
05-09 10:15:10 jdbc[10]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 17 more
05-09 10:15:10 jdbc[10]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 17 more
05-09 10:15:10 jdbc[10]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:363)
	at com.jolbox.bonecp.BoneCP.obtainInternalConnection(BoneCP.java:269)
	at com.jolbox.bonecp.ConnectionHandle.<init>(ConnectionHandle.java:242)
	at com.jolbox.bonecp.PoolWatchThread.fillConnections(PoolWatchThread.java:115)
	at com.jolbox.bonecp.PoolWatchThread.run(PoolWatchThread.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 17 more
05-09 10:54:22 jdbc[13]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at play.utils.ProxyDriver.connect(ProxyDriver.scala:9)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:245)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:243)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:243)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18)
	at play.api.Play$.start(Play.scala:87)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 44 more
05-09 10:54:22 jdbc[13]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at play.utils.ProxyDriver.connect(ProxyDriver.scala:9)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:245)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:243)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:243)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18)
	at play.api.Play$.start(Play.scala:87)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 44 more
05-23 10:11:57 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "CUBRIKPASSWORD" not found; SQL statement:
INSERT INTO AUTHTOKEN (ID,USER, PROVIDER, TOKEN, CUBRIKPASSWORD, CUBRIKTOKEN) VALUES (?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.AuthToken.updateStatus(AuthToken.java:103)
	at service.MyUserServicePlugin.update(MyUserServicePlugin.java:99)
	at com.feth.play.module.pa.PlayAuthenticate.storeUser(PlayAuthenticate.java:167)
	at com.feth.play.module.pa.PlayAuthenticate.loginAndRedirect(PlayAuthenticate.java:393)
	at com.feth.play.module.pa.PlayAuthenticate.handleAuthentication(PlayAuthenticate.java:557)
	at com.feth.play.module.pa.controllers.Authenticate.authenticate(Authenticate.java:25)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$15$$anonfun$apply$15.apply(routes_routing.scala:301)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$15$$anonfun$apply$15.apply(routes_routing.scala:301)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-23 10:21:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Column "CUBRIKPASSWORD" not found; SQL statement:
INSERT INTO AUTHTOKEN (ID,USER, PROVIDER, TOKEN, CUBRIKPASSWORD, CUBRIKTOKEN) VALUES (?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.AuthToken.updateStatus(AuthToken.java:103)
	at service.MyUserServicePlugin.update(MyUserServicePlugin.java:99)
	at com.feth.play.module.pa.PlayAuthenticate.storeUser(PlayAuthenticate.java:167)
	at com.feth.play.module.pa.PlayAuthenticate.loginAndRedirect(PlayAuthenticate.java:393)
	at com.feth.play.module.pa.PlayAuthenticate.link(PlayAuthenticate.java:388)
	at controllers.Account.doLink(Account.java:224)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$32$$anonfun$apply$32.apply(routes_routing.scala:437)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$32$$anonfun$apply$32.apply(routes_routing.scala:437)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.SubjectPresentAction.execute(SubjectPresentAction.java:68)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-23 10:27:02 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "CUBRIKPASSWORD" not found; SQL statement:
INSERT INTO AUTHTOKEN (ID,USER, PROVIDER, TOKEN, CUBRIKPASSWORD, CUBRIKTOKEN) VALUES (?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.AuthToken.updateStatus(AuthToken.java:103)
	at service.MyUserServicePlugin.update(MyUserServicePlugin.java:99)
	at com.feth.play.module.pa.PlayAuthenticate.storeUser(PlayAuthenticate.java:167)
	at com.feth.play.module.pa.PlayAuthenticate.loginAndRedirect(PlayAuthenticate.java:393)
	at com.feth.play.module.pa.PlayAuthenticate.link(PlayAuthenticate.java:388)
	at controllers.Account.doLink(Account.java:224)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$32$$anonfun$apply$32.apply(routes_routing.scala:437)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$32$$anonfun$apply$32.apply(routes_routing.scala:437)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.SubjectPresentAction.execute(SubjectPresentAction.java:68)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-23 10:32:42 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "CUBRIKPASSWORD" not found; SQL statement:
INSERT INTO AUTHTOKEN (ID,USER, PROVIDER, TOKEN, CUBRIKPASSWORD, CUBRIKTOKEN) VALUES (?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.AuthToken.updateStatus(AuthToken.java:103)
	at service.MyUserServicePlugin.update(MyUserServicePlugin.java:99)
	at com.feth.play.module.pa.PlayAuthenticate.storeUser(PlayAuthenticate.java:167)
	at com.feth.play.module.pa.PlayAuthenticate.loginAndRedirect(PlayAuthenticate.java:393)
	at com.feth.play.module.pa.PlayAuthenticate.link(PlayAuthenticate.java:388)
	at controllers.Account.doLink(Account.java:224)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$32$$anonfun$apply$32.apply(routes_routing.scala:437)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$32$$anonfun$apply$32.apply(routes_routing.scala:437)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.SubjectPresentAction.execute(SubjectPresentAction.java:68)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-23 10:34:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "AUTHTOKEN[*] SELECT * FROM LINKED_ACCOUNT"; expected "ALTER, ANALYZE, {"; SQL statement:
AUTHTOKEN SELECT * FROM LINKED_ACCOUNT [42001-172]
05-30 12:34:57 jdbc[7]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at play.utils.ProxyDriver.connect(ProxyDriver.scala:9)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:245)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:243)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:243)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18)
	at play.api.Play$.start(Play.scala:87)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 44 more
05-30 12:34:57 jdbc[7]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at play.utils.ProxyDriver.connect(ProxyDriver.scala:9)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:215)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:245)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:243)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:243)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18)
	at play.api.Play$.start(Play.scala:87)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 44 more
05-30 16:03:27 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
INSERT INTO OTHERUSERINFO (ID,USER, JOBPOSITION, MAININTERESTS, AREAOFSPECIALIZATION, YOURLINKEDIN, YOURRESERCHERGATE, COMPANYHOMEPAGE, YOURHOMEPAGE, ABOUT) VALUES (?,?,?,?,?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.create(OtherUserInfo.java:70)
	at models.User.create(User.java:227)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:161)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:31)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.authenticate(UsernamePasswordAuthProvider.java:87)
	at com.feth.play.module.pa.PlayAuthenticate.handleAuthentication(PlayAuthenticate.java:438)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.handleSignup(UsernamePasswordAuthProvider.java:146)
	at controllers.Signup.doSignup(Signup.java:77)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:237)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:237)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 16:28:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
INSERT INTO OTHERUSERINFO (ID,USER, JOBPOSITION, MAININTERESTS, AREAOFSPECIALIZATION, YOURLINKEDIN, YOURRESERCHERGATE, COMPANYHOMEPAGE, YOURHOMEPAGE, ABOUT) VALUES (?,?,?,?,?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.create(OtherUserInfo.java:70)
	at models.User.create(User.java:227)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:161)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:31)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.authenticate(UsernamePasswordAuthProvider.java:87)
	at com.feth.play.module.pa.PlayAuthenticate.handleAuthentication(PlayAuthenticate.java:438)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.handleSignup(UsernamePasswordAuthProvider.java:146)
	at controllers.Signup.doSignup(Signup.java:77)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:237)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:237)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 16:31:26 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
INSERT INTO OTHERUSERINFO (ID,USER, JOBPOSITION, MAININTERESTS, AREAOFSPECIALIZATION, YOURLINKEDIN, YOURRESERCHERGATE, COMPANYHOMEPAGE, YOURHOMEPAGE, ABOUT) VALUES (?,?,?,?,?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.create(OtherUserInfo.java:70)
	at models.User.create(User.java:227)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:161)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:31)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.authenticate(UsernamePasswordAuthProvider.java:87)
	at com.feth.play.module.pa.PlayAuthenticate.handleAuthentication(PlayAuthenticate.java:438)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.handleSignup(UsernamePasswordAuthProvider.java:146)
	at controllers.Signup.doSignup(Signup.java:77)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:237)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:237)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:07:57 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
INSERT INTO OTHERUSERINFO (ID,USER, JOBPOSITION, MAININTERESTS, AREAOFSPECIALIZATION, YOURLINKEDIN, YOURRESERCHERGATE, COMPANYHOMEPAGE, YOURHOMEPAGE, ABOUT) VALUES (?,?,?,?,?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.save(OtherUserInfo.java:74)
	at models.OtherUserInfo.setJob(OtherUserInfo.java:161)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:08:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
INSERT INTO OTHERUSERINFO (ID,USER, JOBPOSITION, MAININTERESTS, AREAOFSPECIALIZATION, YOURLINKEDIN, YOURRESERCHERGATE, COMPANYHOMEPAGE, YOURHOMEPAGE, ABOUT) VALUES (?,?,?,?,?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.save(OtherUserInfo.java:74)
	at models.OtherUserInfo.setJob(OtherUserInfo.java:161)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:22:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJob(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:22:53 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJob(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:23:01 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJob(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:32:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:32:36 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:32:40 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:33:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:33:53 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:34:19 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
UPDATE OTHERUSERINFO SET ID = ?, JOBPOSITION = ?, MAININTERESTS = ?, AREAOFSPECIALIZATION = ?, YOURLINKEDIN = ?, YOURRESERCHERGATE = ?, COMPANYHOMEPAGE = ?, YOURHOMEPAGE = ?, ABOUT = ? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:44:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "SET" not found; SQL statement:
UPDATE OTHERUSERINFO SET set job_position=?, main_interests=?, area_of_specialization=?, your_linkedin=?, your_researcher_gate=?, company_homepage=?, your_homepage=?, about=? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:45:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "SET" not found; SQL statement:
UPDATE OTHERUSERINFO SET set job_position=?, main_interests=?, area_of_specialization=?, your_linkedin=?, your_researcher_gate=?, company_homepage=?, your_homepage=?, about=? WHERE USER=? [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.readTableColumn(Parser.java:655)
	at org.h2.command.Parser.parseUpdate(Parser.java:689)
	at org.h2.command.Parser.parsePrepared(Parser.java:427)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:168)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:202)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 17:55:33 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Column "JOBPOSITION" not found; SQL statement:
INSERT INTO OTHERUSERINFO (ID,USER, JOBPOSITION, MAININTERESTS, AREAOFSPECIALIZATION, YOURLINKEDIN, YOURRESERCHERGATE, COMPANYHOMEPAGE, YOURHOMEPAGE, ABOUT) VALUES (?,?,?,?,?,?,?,?,?,?) [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:613)
	at org.h2.command.Parser.parseColumn(Parser.java:802)
	at org.h2.command.Parser.parseColumnList(Parser.java:787)
	at org.h2.command.Parser.parseInsert(Parser.java:975)
	at org.h2.command.Parser.parsePrepared(Parser.java:375)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at com.jolbox.bonecp.ConnectionHandle.prepareStatement(ConnectionHandle.java:1024)
	at models.OtherUserInfo.saveInfo(OtherUserInfo.java:74)
	at models.OtherUserInfo.createInfo(OtherUserInfo.java:42)
	at models.User.create(User.java:227)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:161)
	at providers.MyUsernamePasswordAuthProvider.signupUser(MyUsernamePasswordAuthProvider.java:31)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.authenticate(UsernamePasswordAuthProvider.java:87)
	at com.feth.play.module.pa.PlayAuthenticate.handleAuthentication(PlayAuthenticate.java:438)
	at com.feth.play.module.pa.providers.password.UsernamePasswordAuthProvider.handleSignup(UsernamePasswordAuthProvider.java:146)
	at controllers.Signup.doSignup(Signup.java:77)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:241)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$5$$anonfun$apply$5.apply(routes_routing.scala:241)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 18:08:34 jdbc[27]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "10" for parameter "parameterIndex" [90008-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1338)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:347)
	at com.jolbox.bonecp.PreparedStatementHandle.setString(PreparedStatementHandle.java:1013)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:177)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:200)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 18:17:12 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "10" for parameter "parameterIndex" [90008-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1338)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:347)
	at com.jolbox.bonecp.PreparedStatementHandle.setString(PreparedStatementHandle.java:1013)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:117)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:140)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 18:18:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "10" for parameter "parameterIndex" [90008-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1338)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:347)
	at com.jolbox.bonecp.PreparedStatementHandle.setString(PreparedStatementHandle.java:1013)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:117)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:140)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
05-30 18:21:41 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "10" for parameter "parameterIndex" [90008-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1338)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:347)
	at com.jolbox.bonecp.PreparedStatementHandle.setString(PreparedStatementHandle.java:1013)
	at models.OtherUserInfo.updateInfo(OtherUserInfo.java:117)
	at models.OtherUserInfo.setJobPosition(OtherUserInfo.java:140)
	at controllers.Account.modifyJobPosition(Account.java:111)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$8$$anonfun$apply$8.apply(routes_routing.scala:265)
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183)
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56)
	at play.GlobalSettings$1.call(GlobalSettings.java:64)
	at be.objectify.deadbolt.java.actions.RestrictAction.applyRestriction(RestrictAction.java:59)
	at be.objectify.deadbolt.java.actions.AbstractRestrictiveAction.execute(AbstractRestrictiveAction.java:52)
	at be.objectify.deadbolt.java.actions.AbstractDeadboltAction.call(AbstractDeadboltAction.java:119)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91)
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
